### PR TITLE
Organize code

### PR DESCRIPTION
**Pull request by rhnonose**
- Remove some unused code
- Add vivareal.com.br domain on seeds to version the selectors/parsers/filters
- Fix some hidden bugs

### CODE REVIEW

File: priv/repo/seeds.exs [hunk 1]
```diff
@@ -6,38 +6,65 @@ alias Hammock.{
...
 }
 
-Repo.delete_all(ScrapingPath)
 Repo.delete_all(EntryPoint)
 Repo.delete_all(CrawledLink)
 Repo.delete_all(Domain)
 
-em_casa_selectors = %Selectors{
-  price: "span.price span"
+# emcasa_selectors = %Selectors{
```
Comment (gusaiani) [inline priv/repo/seeds.exs]: @rhnonose What of these commented lines?

Comment (rhnonose) [reply priv/repo/seeds.exs]: Just to keep as a reference

Comment (gusaiani) [reply priv/repo/seeds.exs]: @rhnonose Are you going to use these very soon?
Why not leave them in your note pad or somewhere outside the main code?

Comment (rhnonose) [reply priv/repo/seeds.exs]: Probably not using, I'll remove then.

**Pull request merged**